tests/ts_calibrate.c

```diff
@@ -160,6 +161,11 @@ static int ts_validate(struct tsdev *ts, int boundary, unsigned int loops, int t
 
 	if (loops == 0)
 		loops = VALIDATE_LOOPS_DEFAULT;
+	if (edgebuf < EDGE_BUFFER_MIN || (edgebuf + 1) > ((int)xres/2) || (edgebuf + 1) > ((int)yres/2)) {
+		edgebuf = EDGE_BUFFER_MIN;
+		fprintf(stderr, "Edge buffer out of range. Using %d\n",
+			edgebuf);
+	}
 
```

Comment:
why the cast to int here? anyways: gcc warns here, which we want to avoid:

```
ts_calibrate.c: In function ‘ts_validate’:
ts_calibrate.c:164:14: warning: comparison of unsigned expression < 0 is always false [-Wtype-limits]
  if (edgebuf < EDGE_BUFFER_MIN || (edgebuf + 1) > ((int)xres/2) || (edgebuf + 1) > ((int)yres/2)) {
              ^
ts_calibrate.c:164:49: warning: comparison of integer expressions of different signedness: ‘unsigned int’ and ‘int’ [-Wsign-compare]
  if (edgebuf < EDGE_BUFFER_MIN || (edgebuf + 1) > ((int)xres/2) || (edgebuf + 1) > ((int)yres/2)) {
                                                 ^
ts_calibrate.c:164:82: warning: comparison of integer expressions of different signedness: ‘unsigned int’ and ‘int’ [-Wsign-compare]
  if (edgebuf < EDGE_BUFFER_MIN || (edgebuf + 1) > ((int)xres/2) || (edgebuf + 1) > ((int)yres/2)) {
```